### PR TITLE
GP2 220 - Update Dashboard tools and tutorials

### PR DESCRIPTION
**Pull request by csbarbosa**
(Ticket:https://asaphub.atlassian.net/browse/GP2-220)
Update the tool text and links

### CODE REVIEW

File: packages/gp2-components/src/templates/DashboardPageBody.tsx [hunk 1]
```diff
@@ -66,7 +67,7 @@ const DashboardPageBody: React.FC<DashboardPageBodyProps> = ({
                   </Paragraph>
                   <ExternalLink
```
Comment (jhonny-m) [inline packages/gp2-components/src/templates/DashboardPageBody.tsx:68]: lets delete this one for now

File: packages/gp2-components/src/templates/DashboardPageBody.tsx [hunk 2]
```diff
@@ -128,19 +129,67 @@ const DashboardPageBody: React.FC<DashboardPageBodyProps> = ({
...
+              title: 'Request a New Project',
               description: (
                 <>
                   <Paragraph>
-                    In support of the ASAP strategic objectives of supporting
-                    collaborations, generating resources, and democratizing
-                    access to data, GP2 has been established as a
-                    pre-competitive consortium. GP2’s intellectual property (IP)
-                    policy is briefly summarized here.
+                    If you are interest in starting a new project in the GP2 Hub
```
Comment (dianaguimaraes) [inline packages/gp2-components/src/templates/DashboardPageBody.tsx:171]: This copy doesn't seem correct. It should be `interested`, right?

File: packages/gp2-components/src/templates/DashboardPageBody.tsx [hunk 3]
```diff
@@ -128,19 +129,67 @@ const DashboardPageBody: React.FC<DashboardPageBodyProps> = ({
...
+                  <article>
+                    <Subtitle>IP Policy</Subtitle>
+                    <Paragraph>
+                      In support of the ASAP strategic objectives of supporting
+                      collaborations, generating resources, and democratizing
+                      access to data, GP2 has been established as a
+                      pre-competitive consortium. GP2’s intellectual property
+                      (IP) policy is briefly summarized here.
+                    </Paragraph>
+                    <ExternalLink
```
Comment (jhonny-m) [inline packages/gp2-components/src/templates/DashboardPageBody.tsx:144]: this link is wrong, maybe delete it for now

Comment (jhonny-m) [inline packages/gp2-components/src/templates/DashboardPageBody.tsx:144]: where did you get this link from?

File: packages/gp2-components/src/templates/DashboardPageBody.tsx (hunk 2 above)
Comment (csbarbosa) [reply packages/gp2-components/src/templates/DashboardPageBody.tsx:171]: Good call, in the designs was wrong and didn't check! Thank you

File: packages/gp2-components/src/templates/DashboardPageBody.tsx [hunk 4]
```diff
@@ -128,19 +129,67 @@ const DashboardPageBody: React.FC<DashboardPageBodyProps> = ({
...
+                      href="https://hub.asap.science/discover/tutorials/d9c82f68-3f43-4dd8-83c0-179592fc8e42"
```
Comment (jhonny-m) [inline packages/gp2-components/src/templates/DashboardPageBody.tsx:145]: https://gp2.org/resources/intellectual-property-policy/ correct link

**Review by dianaguimaraes — APPROVED** [review]

**Pull request merged**